test/jdk/java/awt/datatransfer/Independence/IndependenceAWTTest.java

```diff
@@ -136,37 +160,35 @@ public void compareText (boolean mustEqual) {
         }
     }
 
-    public void doTest() throws Exception {
+    private static void test() throws Exception {
         getPrimaryClipboard();
-        ExtendedRobot robot = new ExtendedRobot();
-        robot.waitForIdle(1000);
-        frame.setLocation(100, 100);
-        robot.waitForIdle(1000);
+        robot.waitForIdle(500);
 
-        if (pClip != null) {
-            Point ttf1Center = tf1.getLocationOnScreen();
-            ttf1Center.translate(tf1.getWidth()/2, tf1.getHeight()/2);
+        if (primaryClip != null) {
```

Comment:
Do we even need this check? I mean in the getPrimaryClipboard() you throw a runtime exception if primaryClip is null. The only possible option when there is no exception thrown and primaryClip is null is when inside getPrimaryClipboard() HeadlessException happened which should not be the case because of headful tag in the header. So what are we checking for here? In original test it was a soft way to skip test on Windows and macOS but you eliminated that check and added platform requirement to the header.

Reply:
That's correct, the check is redundant and no longer needed after the addition of `@requires` tag. Removed it in the latest commit.